app/javascript/application.js

```diff
@@ -0,0 +1,3 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import "@hotwired/turbo-rails"
```

Comment:
Why was the change needed?

Reply:
Apu, this is a JavaScript package that integrates Turbo with Rails application. Basically using it for more efficient navigation without requiring full page reloads.